### ISSUE TITLE
How did you create the header.png?

### ISSUE THREAD

**Issue by mattmacy** [bamos/densenet.pytorch#2]
I'm quite impressed with how you've presented your densenet implementation. 

V-Net  as a bit messier in terms of needing substantial preprocessing of the data set, a custom loader, and a custom loss function. Nonetheless, I'm patterning the presentation of my implementation https://github.com/mattmacy/vnet.pytorch after yours and I'm wondering how you created the header.png image. 

Thanks in advance..

**Comment by bamos** [comment]
Thanks! I created that image with Keynote. I'd love to see your finished repo, ping me when it's finished.

-Brandon.

**Comment by mattmacy** [comment]
@bamos I'm noticing that, depending on the seed, all voxels are initially attributed to either background or the segmentation object (I'm starting out by testing lungs as that gives me a much better class balance than segmenting nodules), with the softmax values typically around 0.75/0.25. I see them *slowly* moving toward 0.5/0.5 as each batch is processed. Clearly the default weight initialization leaves something to be desired. There are tensor initialization routines, but initializing the whole network is bespoke and bit clumsy. Is there any possibility that that could have influenced your final error value? In other words, with better initialization might you have gotten closer to the paper's result?
 
Cheers.
Matt
